WalletWasabi.Fluent/Views/Wallets/MusicBoxCoinjoinProfileSelector.axaml

```diff
@@ -0,0 +1,164 @@
+      <Flyout Placement="Top" ShowMode="Transient">
+        <DockPanel>
+          <TextBlock Text="Select your profile" Margin="0 5 0 5"
+                     DockPanel.Dock="Top" />
+          <ItemsControl Items="{Binding Profiles}">
+            <ItemsControl.Styles>
+              <Style Selector="Viewbox">
+                <Setter Property="DockPanel.Dock" Value="Left" />
+                <Setter Property="Margin" Value="10" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="Stretch" Value="Uniform" />
+              </Style>
+              <Style Selector="Viewbox > PathIcon">
+                <Setter Property="ClipToBounds" Value="False" />
+              </Style>
+            </ItemsControl.Styles>
+
+            <ItemsControl.DataTemplates>
+
+              <!-- Private -->
+              <DataTemplate DataType="profiles:PrivateCoinJoinProfileViewModel">
+                <c:TileButton Command="{Binding $parent[UserControl].DataContext.SelectProfileCommand}"
+                              CommandParameter="{Binding}">
+                  <DockPanel>
+                    <Viewbox>
+                      <PathIcon Data="{StaticResource coinjoin_privacy}" />
+                    </Viewbox>
+
+                    <StackPanel VerticalAlignment="Center" Spacing="2">
+                      <TextBlock Text="Maximum Privacy" TextAlignment="Center" />
+                      <TextBlock Text="Choice of the paranoid" TextAlignment="Center" />
+                    </StackPanel>
+                  </DockPanel>
+                </c:TileButton>
+              </DataTemplate>
+
+              <!-- Speedy -->
+              <DataTemplate DataType="profiles:SpeedyCoinJoinProfileViewModel">
+                <c:TileButton Command="{Binding $parent[UserControl].DataContext.SelectProfileCommand}"
+                              CommandParameter="{Binding}">
+                  <DockPanel>
+                    <Viewbox>
+                      <PathIcon Data="{StaticResource coinjoin_speed}" />
+                    </Viewbox>
+
+                    <StackPanel>
+                      <TextBlock Text="Maximum Speed" TextAlignment="Center" />
+                      <TextBlock Text="Privacy Faster" TextAlignment="Center" />
+                    </StackPanel>
+                  </DockPanel>
+                </c:TileButton>
+              </DataTemplate>
+
+              <!-- Economic -->
+              <DataTemplate DataType="profiles:EconomicCoinJoinProfileViewModel">
+                <c:TileButton Command="{Binding $parent[UserControl].DataContext.SelectProfileCommand}"
+                              CommandParameter="{Binding}">
+                  <DockPanel>
+                    <Viewbox>
+                      <PathIcon Data="{StaticResource coinjoin_cost}" />
+                    </Viewbox>
+
+                    <TextBlock Text="Economic Mode" TextAlignment="Center" VerticalAlignment="Center" />
```

Comment:
```suggestion
                    <TextBlock Text="Minimize cost" TextAlignment="Center" VerticalAlignment="Center" />
```
To be standard with the others.

There should also be a second line with short explanation.